microsite/sidebars.js

```diff
@@ -116,6 +116,66 @@ module.exports = {
           'features/search/how-to-guides',
         ],
       },
+      {
+        type: 'category',
+        label: 'Auth and Identity',
```

Comment:
Let's sort all entries in this section alphabetically

Reply:
updated in [0d6efa4](https://github.com/backstage/backstage/pull/27803/commits/0d6efa49ac7931a721e7fda97766947feead8c62)